src/main/java/sg/ncl/MainController.java

```diff
@@ -1522,7 +1565,78 @@ public String editTeamProfile(
 
         // safer to remove
         session.removeAttribute("originalTeam");
-        return "redirect:/team_profile/" + teamId;
+        return REDIRECT_TEAM_PROFILE + teamId;
+    }
+
+    @RequestMapping(value = "/team_quota/{teamId}", method = RequestMethod.POST)
+    public String editTeamQuota(
+            @PathVariable String teamId,
+            @ModelAttribute("teamQuota") TeamQuota editTeamQuota,
+            final RedirectAttributes redirectAttributes,
+            HttpSession session) throws IOException {
+
+        final String QUOTA = "#quota";
+        JSONObject teamQuotaJSONObject = new JSONObject();
+        teamQuotaJSONObject.put(TEAM_ID, teamId);
+
+        // check if budget is negative or exceeding limit
+        if (!editTeamQuota.getBudget().equals("")) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)